src/Tizen.Applications.Common/Tizen.Applications/CoreApplication.cs

```diff
@@ -277,6 +284,17 @@ protected virtual void OnDeviceOrientationChanged(DeviceOrientationEventArgs e)
             DeviceOrientationChanged?.Invoke(this, e);
         }
 
+        /// <summary>
+        /// Overrides this method if want to handle behavior when the time zone is changed.
```

Comment:
Overrides -> Override, "if want" -> "if you want"
 I see that the same is in other On* methods

Reply:
Done